docker/puppetdb/docker-entrypoint.sh

```diff
@@ -11,10 +11,10 @@ if [ ! -d "/etc/puppetlabs/puppetdb/ssl" ] && [ "$USE_PUPPETSERVER" = true ]; th
     sleep 1
   done
   set -e
-  /opt/puppetlabs/bin/puppet config set certname "$HOSTNAME"
-  /opt/puppetlabs/bin/puppet config set server "$PUPPETSERVER_HOSTNAME"
-  /opt/puppetlabs/bin/puppet agent --verbose --onetime --no-daemonize --waitforcert 120
-  /opt/puppetlabs/server/bin/puppetdb ssl-setup -f
+  /ssl.sh
```

Comment:
This one line replaces the 4 lines above it.

The script is hardcoded to:
* use `$HOSTNAME` (overridable via argument) for the cert subject and filename
* use `$PUPPETSERVER_HOSTNAME` (overridable via ENV variable though), 
* wait 120 seconds for the cert to be signed (overridable via ENV variable though) 

In other words these two invocations are the same:

```bash
/ssl.sh
PUPPETSERVER_HOSTNAME="$PUPPETSERVER_HOSTNAME" WAITFORCERT=120 /ssl.sh "$HOSTNAME"
```